### PR TITLE
feat(button): update EU button styles - FRONT-4323

### PR DESCRIPTION
**Pull request by emeryro**
- icon size is now `xs` by default (same as EC), and changes to `m` when there is no label (handled via css)
- different display for active state
- background, font color and border color updated

### CODE REVIEW

File: src/themes/ec/variables/_button.scss [hunk 1]
```diff
@@ -3,11 +3,13 @@
 
 $button: (
   radius: 2px,
+  font: var(--f-m),
   shadow: none,
   border-width: 2px,
   outline-width: 1px,
   padding-horizontal: var(--s-m),
   padding-vertical: var(--s-xs),
+  icon-only-size: map.get($icon, 'xs'),
```
Comment (planctus) [inline src/themes/ec/variables/_button.scss]: i wonder why we are notsimply doing the same in EC and EU, we aligned now the default icon size but we introduced this difference in the icon only use case..

File: src/implementations/twig/components/button/button.story.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ const prepareData = (data, args) => {
   if (args.icon_name && args.icon_name !== 'none') {
     data.icon = {};
     data.icon.name = args.icon_name;
-    data.icon.size = system === 'eu' ? 's' : 'xs';
+    data.icon.size = 'xs';
```
Comment (planctus) [inline src/implementations/twig/components/button/button.story.js:150]: Can we make the hide_label control conditional? It should appear only when an icon has been chosen

File: src/themes/ec/variables/_button.scss (hunk 1 above)
Comment (emeryro) [reply src/themes/ec/variables/_button.scss]: I'll check with the designers

File: src/implementations/twig/components/button/button.story.js (hunk 2 above)
Comment (emeryro) [reply src/implementations/twig/components/button/button.story.js:150]: Well, can we do that? :)
As far as I understood, the conditional control (with 'if') only work with boolean, I don't know if we can detect if a select list has a specific value

Comment (emeryro) [reply src/implementations/twig/components/button/button.story.js:150]: I updated the story to set a default icon if the hide_label control is true.
That's not perfect, but already better

Comment (planctus) [reply src/implementations/twig/components/button/button.story.js:150]: but we have already in the button story file the same condition used for the icon args:
`if: { arg: 'icon_name', neq: 'none' },`

Comment (emeryro) [reply src/implementations/twig/components/button/button.story.js:150]: that's true... I'll update

**Review by planctus — APPROVED** [review]

**Pull request merged**